my-app/src/utils/Valid.ts

```diff
@@ -1,9 +1,11 @@
-import { IUserRegister, IBlog } from './Typescript'
+import { Email } from '@material-ui/icons';
+import { IUserRegister } from './Typescript'
 
 export const validRegister = (userRegister: IUserRegister) => {
-  const { name, account, password, cf_password } = userRegister;
+  const { name, account, password } = userRegister;
   const errors: string[] = [];
 
+  
   if(!name){
     errors.push("Please add your name.")
   }else if(name.length > 20){
```

Comment:
I think we should take this out since putting a restriction on name length doesn't make sense to me